### PR TITLE
Add issue and pull request templates

### LINKED ISSUE #367: Add issue template

### PR DESCRIPTION
**Pull request by mpenkov**
- Fix #367

### CODE REVIEW

**Comment by mpenkov** [comment]
I've covered the bases fairly well. Please let me know if there are things you'd like to adjust.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+#### Title
+
+Please pick a concise, informative and complete title for your PR.
```
Comment (piskvorky) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
Please **pick a concise, informative and complete title** for your PR.
```

Comment (piskvorky) [reply .github/PULL_REQUEST_TEMPLATE.md]: I cannot push to this PR - just adding **bold** here.

**Comment by piskvorky** [comment]
Looks awesome, LGTM!

Better than Gensim's in fact.

**Comment by piskvorky** [comment]
@mpenkov the build is failing now (the build badge on main README page is red again).

**Pull request merged**